.github/workflows/cla.yml

```diff
@@ -0,0 +1,36 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened,closed,synchronize]
+
+jobs:
+  CLAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "CLA Assistant"
+        if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
+        # Beta Release
+        uses: cla-assistant/github-action@v2.1.3-beta
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # the below token should have repo scope and must be manually added by you in the repository's secret
+          PERSONAL_ACCESS_TOKEN : ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+        with:
+          path-to-signatures: 'signatures/version1/cla.json'
+          path-to-document: 'https://gist.github.com/stsrki/abfa5ce0f4a5cf1e6ac67b92f8eb5d63' # e.g. a CLA or a DCO document
+          # branch should not be protected
+          branch: 'master'
```

Comment:
You're right. Maybe it allows multiple values? I'll have to investigate.
Also I just noticed: "#branch should not be protected"... No idea if this will have a different behaviour on Blazorise master branch

Reply:
WEll it seems it still runs successfully as long as the workflow is still there, so we should also merge this in to rel-1.0 branch.

![image](https://user-images.githubusercontent.com/22283161/161165892-4d18c1f4-c4e9-44ec-9aae-b78e4b849f48.png)

Reply:
We can cherry-pick the commit into rel-1.0 branch if needed.